projects/batfish/src/test/resources/org/batfish/grammar/cisco/testrigs/ibgp-reject-routerid-match/configs/r3

```diff
@@ -0,0 +1,23 @@
+! boot system flash:EOS.swi
+hostname r3
+!
+interface Ethernet1
+   no switchport
+   ip address 10.34.0.3/24
+!
+interface Loopback0
+   ip address 3.0.0.1/32
+!
+ip routing
+!
+router bgp 1
+!!! same ID as r1
+   router-id 9.9.9.9
+   neighbor 10.34.0.4 remote-as 1
```

Comment:
Please use loopbacks to establish iBGP sessions. Add static routes to those loopbacks wherever they are referred to. The next hop should be the physical interface address of the neighbor.